spec/requests/providers_spec.rb

```diff
@@ -935,6 +948,17 @@ def failed_auth_action(id)
       expect_single_action_result(failed_auth_action(provider.id.to_s).symbolize_keys)
     end
 
+    it "supports cloud provider refresh" do
+      api_basic_authorize collection_action_identifier(:providers, :refresh)
```

Comment:
I think for this we want the specific identifier, maybe api_basic_authorize('ems_cloud_refresh')